tensorrt_llm/_torch/model_config.py

```
@@ -425,7 +425,32 @@ def _build_modelopt_quant_config(json_quant_configs, checkpoint_dir,
+                (_MODEL_PREFIX + k[len(_LM_PREFIX):] if k.startswith(_LM_PREFIX) else k): v
+                for k, v in mixed_quant_configs.items()
+            }
+            # LMHead bypasses Linear.create_weights (manual Parameter),
+            # so NVFP4 weight scales are never allocated there.  Move
+            # lm_head to exclude_modules so it loads as BF16.
+            if mixed_quant_configs and "lm_head" in mixed_quant_configs:
+                if quant_config.exclude_modules is None:
+                    quant_config.exclude_modules = []
+                if "lm_head" not in quant_config.exclude_modules:
+                    quant_config.exclude_modules = list(
+                        quant_config.exclude_modules) + ["lm_head"]
```

Ruff (0.15.18)

[warning] Consider `[*list(quant_config.exclude_modules), "lm_head"]` instead of concatenation

Replace with `[*list(quant_config.exclude_modules), "lm_head"]`

(RUF005)